gcp-firebase/Dockerfile

```diff
@@ -1,17 +1,20 @@
 FROM google/cloud-sdk:252.0.0-slim
 
+ARG FIREBASE_VERSION=7.3.1
+ARG SOPS_VERSION=3.3.1
+
 # Installs NodeJS
 RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
-       && apt-get install -y nodejs
+    && apt-get install -y nodejs
 
 # Installs firebase
-RUN npm install -g firebase-tools@7.0.2
+RUN npm install -g firebase-tools@${FIREBASE_VERSION}
 
 # Installs SOPS
 RUN SOPS_PKG=/tmp/sops/sops.deb \
-    && SOPS_SOURCE=https://github.com/mozilla/sops/releases/download/3.2.0/sops_3.2.0_amd64.deb \
+    && SOPS_SOURCE=https://github.com/mozilla/sops/releases/download/${SOPS_VERSION}/sops_${SOPS_VERSION}_amd64.deb \
```

Comment:
You can actually use [lsb_release](https://refspecs.linuxfoundation.org/LSB_3.0.0/LSB-PDA/LSB-PDA/lsbrelease.html) to make this portable beyond the `amd64` platform.  We don't need to change the script for as this looks to work just fine, worth remembering for other projects though.

Reply:
Thanks, I was aware of it however I thought it was best to keep my changes minimal due to my time constraint, I'll definitely keep it in mind for future though!